doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+
+# Introduction 
+
+The number of indigenous incarcerations in Canada has increased by 40% in the prison system over the last decade, a faster rate than any other group [@ owusu2014years].  Issues such as increased domestic violence, disruptions in societal structure and norms and health issues are aggravated in highly incarcerated groups [@ freudenberg2001jails]. Hence, these rates are likely to worsen.  
+
+As this population is so vulnerable, the other key factor that should be investigated is if there is a difference in the length of the sentence between the indigenous and non-indigenous groups. This is an important question to infer as it can lead to further studies on the type of charges either group is subject too and whether particular laws disadvantage the indigenous group if a significant difference is found. Additionally, the study could also lead to an exploration of biases present in the current legal system when sentencing. 
+
+# Methods 
+
+## Data 
+
+The data set used for this study is the Offender Profile from 2017-2018 released by the Correctional Service of Canada. The link to this site can be found [here](https://open.canada.ca/data/en/dataset/844ff1e3-e137-41be-9ebe-6bd9843c1a53). Each entry in the dataset corresponds to a single offender serving a two or more year long sentence. The demographic details such as age, gender and marital status at year end are provided for each entry. This was retrieved from the Offender Management System (OMS).
+
+##  Analysis
+
+A hypothesis test with a significance level of 0.05 was carried out to determine if there was a statistically significant difference in the medians of the sentence length between the indigenous and non-indigenous groups. The features used from the data set to complete this analysis were the racial grouping, which was already separated into indigenous and non-indigenous, and aggregate sentence length. The median was selected as the measure of central tendency and the permutation test under the null hypothesis. The null hypothesis stated that there was no difference in the population medians in the sentence length between indigenous and non-indigenous inmates. The alternate hypothesis stated that there was a difference in the population median between indigenous and non-indigenous inmates.
+
+The R and Python programming languages [@R; @Python] and the following R and Python packages were used to perform the analysis:  docopt [@docopt],  knitr [@knitr], tidyverse [@tidyverse], infer[@infer], docopt [@docoptpython]. The code used to perform the analysis and create this report can be found here: https://github.com/UBC-MDS/DSCI_522_inference_on_indigenous_vs_non_indigenous_sentence_length_differences.
```

Comment:
also altair

Reply:
I just finished the hypothesis test and get this conclusion. We can add it after you all reviewed my work.

Conclusion: Since the test statistic falls in the left tail in the distribution plot and the $p$-value $= 0.0328 < 0.05$, we can conclude that given the significant level of $\alpha = 0.05$, there is statistically significant evidence that we can reject $H_0$ such that there is significant difference in the length of sentences between indigenous and non-indigenous offenders in the Correctional Services of Canada.

Reply:
Noted and changed

Reply:
I think it will be good to specify that we conducted a two-tailed hypothesis test in our conclusion - I will add it in.